### DISCUSSION TITLE
Welcome to ha-porscheconnect Discussions!

### DISCUSSION THREAD

**Discussion by CJNE** [CJNE/ha-porscheconnect#255] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CJNE** [comment]
Hello everyone! 

Thanks to the tremendous work that @fredriklj has put in to blow new life into this project it looks like we're going to have a working solution for communicating with the Porsche APIs again, despite the setbacks we've experienced.

To celebrate this we're opening up the discussions space where we can discuss more freely around things that are strictly not related to a specific issue.

We're currently adding some final issues that needs to be resolved before we can roll out the next release, so hopefully it won't be long now.

**Comment by gadgetbazza** [comment]
Massive thanks to those that worked to resolve the API issues and conducted the testing.  Nice to finally have my car in HA!

A question if I may.  The odometer and milage remaining show in kilometers rather than miles, is there a way I can configure this to show in my preferred units, or do I need to do something more generic in HA for this?  I guess I could create a template sensor to make the conversion, but I thought that HA allowed for this based upon your locale to make the conversions automatically?

**Comment by fredriklj** [comment]
Hi! Yes, HA should set the units according to your locale. But you should also easily be able to change it by clicking the entity and changing its settings.

**Comment by alegend2k** [comment]
Can anyoone tell me why always version 0.1.1 gets installed although 0.1.2 is available? I tried many times but no change...
With 0.1.1. I cant open / lock door because it says pin code not provided...

**Reply by fredriklj** [reply]
It is because I messed up when doing the last release. You have the lastest code installed, it is just the displayed version number that is off.

**Reply by alegend2k** [reply]
ah ok but it keeps sayin "no pin code provided" although i typed it in the box within home assistsant....
any clue?

**Reply by fredriklj** [reply]
Could this be it?

https://github.com/CJNE/ha-porscheconnect/issues/257

In that case it will be included as a fix in the upcoming release.

**Comment by alegend2k** [comment]
Yes that seems to be the problem...

**Comment by tomaszwyka** [comment]
Thanks! Works great with my 911 992👌

**Comment by AbdullahAlacozai** [comment]
Can't seem to login - I can login to my app on my phone but the same credentials will not work with this - any issues with an NZ account?

**Comment by thefunkygibbon** [comment]
this plugin dead?  not seeing many replies from the dev on any of these here discussions etc. 

I don't yet subscribe to connect, but i might do if this works and you can create automations around charging and readings from the car and locks and suchlike.

**Reply by tomaszwyka** [reply]
I have been using this plugin since January this year in my 911 992 and everything works fine

**Reply by fredriklj** [reply]
I'm using it on a daily basis :-)

**Comment by nineknives** [comment]
The front page says this integration does not work with Porsche Car Connect, or Macan models older than the 24/EV range, yet my 2021 ICE Macan shows up perfectly using this integration. I'm able to see lock status, fuel range, when it's home, etc. Just wanted to throw this in in case you didn't know. It's a wonderful integration and I'm super thankful and excited to be able to see things like lock status remotely from my HA dash instead of logging into the My Porsche app. Thank you!
